code/TemplateStudioForUWP/Templates/_catalog/cs-CZ.frontendframeworks.json

```diff
@@ -2,16 +2,16 @@
   {
     "name": "MVVMToolkit",
     "displayName": "MVVM Toolkit",
-    "summary": "Moderní, rychlá, modulární knihovna MVVM a součást sady nástrojů Windows Community Toolkit."
+    "summary": "ModernÃ­, rychlÃ¡, modulÃ¡rnÃ­ knihovna MVVM a souÄÃ¡st sady nÃ¡strojÅ¯ Windows Community Toolkit."
```

Comment:
This looks broken.
What has happened to all these localized files?

Reply:
I'm not sure. I hit encoding issues in the tests, so I ran the script that's supposed to fix all encoding issues. Is https://github.com/microsoft/TemplateStudio/blob/main/_utils/Set-All-Text-File-Template-Encodings.ps1 broken?

Reply:
Also it seems every time I touch a file I get encoding issues. Can we enforce the desired encoding via .editorconfig or some other mechanism so files are always saved with the right encoding?

Reply:
I'm using VS, and VS Code sometimes for some things. I had never opened the json or md files, though, the script just updated them when I ran it. I'll just revert all changes to the json and md files for now.